src/acachecontrol/cache.py

```diff
@@ -30,9 +28,10 @@ class AsyncCache:
     Key: Tuple(http_method, url), value: aiohttp response obj
     """
 
-    def __init__(self, config={}):
+    def __init__(self, config: Dict = None):
```

Comment:
@MasterSergius Why do we need `config` if a user is not able to provide it?

Reply:
Why not able?
Let's assume we have some cache_config:
```py
    async with AsyncCacheControl(cache=AsyncCache(config=cache_config)) as cached_sess:
        async with cached_sess.request('GET', 'http://example.com') as resp:
            resp_text = await resp.text()
            print(resp.status)
            #print(resp.response.headers)
```